src/OSPSuite.Presentation/Presenters/ContextMenus/ContextMenu.cs

```diff
@@ -62,7 +60,7 @@ public abstract class ContextMenu<T> : ContextMenu
       protected ContextMenu(T objectRequestingContextMenu, IContainer container) : base(container)
       {
          _container = container;
-         AllMenuItemsFor(objectRequestingContextMenu).Each(_view.AddMenuItem);
+         AllMenuItemsFor(objectRequestingContextMenu).Where(x => x != null).Each(_view.AddMenuItem);
```

Comment:
@Yuri05 SOmetimes derives class return some items that are null if the entry should not be displayed. It's better to filter them here